CHANGELOG.md

```diff
@@ -6,7 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
-## [7.2.0] - 2022-06-15
+## [7.2.0] - 2022-07-11
+
+### Changed
+- Revert conjurrc key names to match Ruby CLI. **This is a breaking change.**
```

Comment:
Is this still considered a braking change?

Reply:
Good question. I thought maybe in case a user has custom code that depends on reading the conjur_account and conjur_url keys, but I think you're right that it isn't. More of a deprecation.